src/index.ts

```diff
@@ -27,15 +18,18 @@ export const languages: Partial<SupportLanguage>[] = [
 // https://prettier.io/docs/en/plugins.html#parsers
 export const parsers: Record<string, Parser> = {
 	astro: {
-		parse: (source) => parse(source),
+		async parse(source) {
+			const { ast } = await parse(source);
+			return ast;
+		},
```

Comment:
I follow [this ](https://github.com/prettier/prettier/blob/1a602b590ec9ed5db01780939659c427e3d277b8/changelog_unreleased/api/13268.md)

Reply:
Can't this block just be `parse` though? Oh wait you're destructuring, nvm